src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -9754,8 +9754,14 @@ namespace AirflowNetworkBalanceManager {
                 for (j = 1; j <= NumOfZones; ++j) {
                     if (!ZoneEquipConfig(j).IsControlled) continue;
                     if (ZoneEquipConfig(j).ZoneNode == i) {
-                        NodeFound(i) = true;
-                        AirflowNetworkNodeData(ZoneEquipConfig(j).ActualZoneNum).EPlusNodeNum = i;
+                        if (ZoneEquipConfig(j).ActualZoneNum > AirflowNetworkNumOfNodes) {
```

Comment:
OK, so if we find a zone with this matching node number but it's not an AFN distribution node then error out.  Got it.